app/views/katello/api/v2/packages/backend.json.rabl

```diff
@@ -5,3 +5,8 @@ attributes :children, :checksumtype, :size, :url, :build_time, :summary, :group,
 node :human_readable_size do |package|
   number_to_human_size(package.size) if package.size
 end
+
+node :human_readable_time do |package|
+  Time.at(package.build_time).strftime("%B %e, %Y at %I:%M %p") if package.build_time
+end
+
```

Comment:
We can avoid formatting here and use the date components on UI for consistency..Somnething like:
```
<date date = "package.human_readable_time" />
```

Reply:
I think we could probably reference `package.build_time` directly in the UI through the <date> tag

Reply:
That might not work as the date component does not expect epoch time but I am not sure.

Reply:
Ah ok!

Side note: I think human_readable_time should be moved into the ruby code (on the package model) and not in the RABL API template